docs/examples/code/fill_and_submit_web_form_crawler.py

```diff
@@ -1,5 +1,5 @@
 import asyncio
-import json
+import urllib.parse
```

Comment:
A nitpick... Could you please use `from urllib.parse import urlencode` so that it's shorter in the code? Same for the other example. Thanks.